build.gradle.kts

```diff
@@ -353,7 +353,7 @@ tasks.register("javaioPreCommit") {
   dependsOn(":sdks:java:io:jms:build")
   dependsOn(":sdks:java:io:kafka:build")
   dependsOn(":sdks:java:io:kafka:upgrade:build")
-  dependsOn(":sdks:java:io:kafka:file-aware-factories:build")
```

Comment:
we need to add sdks/java/extensions/kafka-factories to the trigger path of https://github.com/apache/beam/blob/master/.github/workflows/beam_PreCommit_Java_IOs_Direct.yml that's why this wasn't captured